docs/K8SCollectors.md

```diff
@@ -0,0 +1,179 @@
+# **K8S data collectors**
+The Splunk Operator for Kubernetes deploys Splunk Enterprise custom resources across a single/multiple namespace/s. The helper scripts `k8s-splunk-collector.sh` and `k8s-systeminfo-collector.sh` in the [tools directory](https://github.com/splunk/splunk-operator/tree/master/tools/k8s_collectors), collect data from a K8S cluster which runs the Splunk Operator for K8S.
+
+## **Splunk Collector for K8S - k8s-splunk-collector.sh**
```

Comment:
Might be worth implementing some additional data collection things such as - 
1. Collecting data/diag from a particular pod only
2. Collecting data/diag from pods of a CR
3. Collecting information such as configMap data(we have seen many cases where the data in configMap is not correct and this leads to crashes or other issues on Splunk operator which is a bit hard to debug)

Reply:
@gaurav-splunk For points 1 and 2, great points we can consider as enhancements. We do collect data from configmaps(with kubectl describe commands and also -o yaml for the kubectl get commands) with the current implementation of the script